src/Elcodi/Admin/CoreBundle/Resources/public/components/plupload/examples/dump.php

```diff
@@ -15,12 +15,16 @@
 		<th>Name</th>
 		<th>Value</th>
 	</tr>
-	<?php $count = 0; foreach ($_POST as $name => $value) { ?>
-	<tr class="<?php echo $count % 2 == 0 ? 'alt' : ''; ?>">
+	<?php $count = 0; foreach ($_POST as $name => $value) {
```

Comment:
I will not change these files, since they're external to our project, and we will get rid of them as soon as possible. (I do hope so)